numba/core/config.py

```diff
@@ -412,6 +412,20 @@ def avx_default():
         CUDA_PER_THREAD_DEFAULT_STREAM = _readenv(
             "NUMBA_CUDA_PER_THREAD_DEFAULT_STREAM", int, 0)
 
+        # Location of the CUDA include files
+        if IS_WIN32:
+            cuda_path = os.environ.get('CUDA_PATH')
+            if cuda_path:
+                default_cuda_include_path = os.path.join(cuda_path, "include")
+            else:
+                default_cuda_include_path = "cuda_include_not_found"
+        else:
+            default_cuda_include_path = "/usr/local/cuda/include"
```

Comment:
Should this be `os.path.join(['usr', 'local', 'cuda', 'include'])`?

Reply:
Yes, that would be better.